<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeLink</class>
 <widget class="QFrame" name="TimeLink">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_1">
      <attribute name="title">
       <string>通路1</string>
      </attribute>
      <layout class="QVBoxLayout" name="vBox_1" stretch=""/>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>通路2</string>
      </attribute>
      <layout class="QVBoxLayout" name="vBox_2"/>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>通路3</string>
      </attribute>
      <layout class="QVBoxLayout" name="vBox_3"/>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>通路4</string>
      </attribute>
      <layout class="QVBoxLayout" name="vBox_4"/>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>通路5</string>
      </attribute>
      <layout class="QVBoxLayout" name="vBox_5"/>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>通路6</string>
      </attribute>
      <layout class="QVBoxLayout" name="vBox_6"/>
     </widget>
     <widget class="QWidget" name="tab_7">
      <attribute name="title">
       <string>通路7</string>
      </attribute>
      <layout class="QVBoxLayout" name="vBox_7"/>
     </widget>
     <widget class="QWidget" name="tab_8">
      <attribute name="title">
       <string>通路8</string>
      </attribute>
      <layout class="QVBoxLayout" name="vBox_8"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
